core/templates/dev/head/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -131,9 +147,14 @@ angular.module('oppia').directive('storyNodeEditor', [
           };
 
           $scope.updateExplorationId = function(explorationId) {
-            StoryUpdateService.setStoryNodeExplorationId(
-              $scope.story, $scope.getId(), explorationId);
-            $scope.currentExplorationId = explorationId;
+            _isExpIdValid(explorationId).then(function(response) {
```

Comment:
Change "response" to "expIdIsValid"; try to be as descriptive as possible. Is it guaranteed that `response` will be a boolean?

Reply:
Done!

Reply:
You have not answered my question. Please answer explicitly all questions in review comments.

Reply:
@seanlip Yes it is the function _isExpIdValid returns a boolean.